Add Base Debian

notAreYouScared opened the pull request:
Add base Debian image for eggs, mainly for ts3 at the moment.

notAreYouScared committed:
Add base Debian

Add base Debian image for eggs, mainly for ts3 at the moment.

DaneEveritt commented on README.md:
```diff
@@ -26,6 +26,10 @@ is tagged correctly.
 
+* [`alpine`](https://github.com/pterodactyl/yolks/tree/master/alpine)
+  * `ghcr.io/pterodactyl/yolks:alpine`
+* [`alpine`](https://github.com/pterodactyl/yolks/tree/master/debian)
```
debian?

DaneEveritt commented on .github/workflows/debian.yml:
```diff
@@ -0,0 +1,34 @@
+  push:
+    branches:
+      - master
+    paths:
```
Can we merge the debian and alpine builds into a single workflow that uses a matrix to build both?

DaneEveritt commented on debian/Dockerfile:
```diff
@@ -0,0 +1,27 @@
+LABEL       org.opencontainers.image.source="https://github.com/pterodactyl/yolks"
+LABEL       org.opencontainers.image.licenses=MIT
+
+RUN         apt update \
```
This seems like more run layers than necessary, they can all be merged into a single layer.

notAreYouScared committed:
Fix readme / combine workflows / combine dockerfile for debian

Fix readme
combine alpine/debian into single workflow (oses)
combine debian run's into single layer

notAreYouScared committed:
Update README.md
